CHANGELOG.md

```diff
@@ -1,3 +1,38 @@
+# 1.1.3 (2021-05-21)
+## Edge Agent
+### Bug Fixes
+* Update GetModuleLogs method when tail + since + until options are provided. [c1dba55](https://github.com/Azure/iotedge/commit/c1dba55ece869932b5a75ae0de658121dd75e54f)
+* Fix vulnerability issues in docker images. [b286408](https://github.com/Azure/iotedge/commit/b28640847c7d8efd82244d0628a36c63e20e5473)
+
+
+## Edge Hub
+### Bug Fixes
+* Adds SharedAccessSignature to repo with fix for vulnerability. [4fea6e7](https://github.com/Azure/iotedge/commit/4fea6e7750984e278e20e27635eea223c68c1e3f)
+* Close AMQP connection explicitly when no more links (removing links kept tcp level connection). [5be30fb](https://github.com/Azure/iotedge/commit/5be30fb80a69fb36a8de08541568c2d1f161fe2b)
+* Fix vulnerability issues in docker images. [b286408](https://github.com/Azure/iotedge/commit/b28640847c7d8efd82244d0628a36c63e20e5473)
+* Fix edgehub_queue_length metric. [4aab90b](https://github.com/Azure/iotedge/commit/4aab90b0aeab80ee30d1c7984a930e643bf39b65)
+
+
+## Azure Monitor Module
```

Comment:
Title should be "Metrics Collector Module"

Reply:
Also, this module is new for this release, so we should include to the commit that introduced the module to the repo.